app/src/views/Matches.vue

```diff
@@ -5,94 +5,22 @@
-                          </div>
-                          <div class="py-1">
-                            <p
-                              v-if="match.latest_message"
-                              class="caption mb-0"
-                            >{{formatDate(index)}}</p>
-                          </div>
-                        </v-row>
-                        <v-row class="d-flex">
-                          <div v-if="match.latest_message">
-                            <pre
-                              class="body-2 mb-0"
-                              :class="match.other_user.user_id == match.latest_message.user_id ? 'indigo--text font-weight-medium' : undefined"
-                            >{{match.latest_message.content.substr(0, 30)}}...</pre>
-                          </div>
-                          <div v-else>
-                            <p class="mb-0">Get the conversation started!</p>
-                          </div>
-                          <!-- {{match.unread_messages}} -->
-                        </v-row>
-                      </v-col>
-                    </v-row>
-                  </div>
-                </v-card>
-              </v-hover>
-            </div>
-          </div>
-          <div v-else>No matches yet, get swiping!</div>
-        </div>
-        <div v-else>
-          <div v-if="$apollo.queries.matches.loading">
-            <div v-for="i in 5" :key="i">
-              <v-card dark outlined class="mb-2">
-                <div class="px-5">
-                  <v-row class="d-flex">
-                    <v-col style="flex-basis: 1; flex-grow: 0">
-                      <v-skeleton-loader dark type="avatar"></v-skeleton-loader>
-                    </v-col>
-                    <v-col class="py-1">
-                      <v-row class="d-flex justify-space-between">
-                        <div class="py-1">
-                          <v-skeleton-loader dark type="text" style="min-width: 100px"></v-skeleton-loader>
-                        </div>
-                        <div class="py-1">
-                          <v-skeleton-loader dark type="text" style="min-width: 50px"></v-skeleton-loader>
-                        </div>
-                      </v-row>
-                      <v-row>
-                        <v-skeleton-loader dark style="min-width: 100%" type="sentences"></v-skeleton-loader>
-                      </v-row>
-                    </v-col>
-                  </v-row>
-                </div>
-              </v-card>
-            </div>
-          </div>
-          <div v-else>Could not load matches</div>
-        </div>
-      </v-fade-transition>
+      <v-tabs dark v-model="tab" icons-and-text>
+        <v-tab>Two way matches<v-icon>mdi-account-switch</v-icon></v-tab>
+        <v-tab>People who have matched back<v-icon>mdi-account-arrow-left</v-icon></v-tab>
+        <v-tab>People you've matched with<v-icon>mdi-account-arrow-right</v-icon></v-tab>
```

Comment:
@rkarim0598 wasn't sure what to go with on these names

Reply:
Maybe "People you like" and "People who like you"?